docs-ref-conceptual/whats-new-overview.md

```diff
@@ -13,6 +13,12 @@ keywords: Azure CLI, new articles, new references, new samples, announcements
 
 This page highlights new features and upcoming changes for the Azure CLI.
 
+## Multi-factor authentication (MFA)
+
+Microsoft will require MFA for all Azure users in a gradual and methodical rollout. For more information, see the [original Microsoft announcement](https://techcommunity.microsoft.com/t5/core-infrastructure-and-security/microsoft-will-require-mfa-for-all-azure-users/ba-p/4140391) and [June 2024 update](https://techcommunity.microsoft.com/t5/core-infrastructure-and-security/update-on-mfa-requirements-for-azure-sign-in/ba-p/4177584).
```

Comment:
Can we explicitly call out what is in the blog post that service principals and managed identities auth based will not be impacted?